app/models/survey.rb

```diff
@@ -22,4 +22,9 @@ def answer_accepting?(survey_id)
           .where('end_at >= ?', Date.current)
           .exists?(survey_id)
   end
+
+  def answer_ended?(survey_id)
```

Comment:
レビューありがとうございます〜！
引数を取らない実装に変更しました。

また、「受付終了」表記は終了日の終了時刻を過ぎたら表示されてほしいなど、日付だけでなく時刻も加味する必要があると思うので時間が返ってくるTimeクラスの実装に変更しました。

#### 参考
[Time](https://api.rubyonrails.org/classes/Time.html#method-c-current)
>Returns Time.zone.now when Time.zone or config.time_zone are set, otherwise just returns Time.now.

[Date](https://api.rubyonrails.org/classes/Date.html#method-c-current)
>Returns [Time.zone](https://api.rubyonrails.org/classes/Time.html#method-c-zone).today when Time.zone or config.time_zone are set, otherwise just returns Date.today.